### PR TITLE
Update documentation


### PR DESCRIPTION
**Pull request by evrardjp**
The expectations of what to run is unclear.

Clarify the usage of Airship as our official method of deployment.

### CODE REVIEW

**Comment by evrardjp** [comment]
I added the documentation for Airship in the deployment, we should now have 100% of the coverage of the confluence page, we just need extra content from the Airship squad, which can happen on a separate commit.

File: doc/source/deploy-guide/overview.rst [hunk 1]
```diff
@@ -62,13 +62,25 @@ Your environment should be setup with the following:
...
+  .. note::
+     If you are not coming with your own node for
+     `deployer`, this tooling can create one for you in an OpenStack
+     environment. However, this should be treated as experimental.
 
-* The :term:`SES` cluster must run :term:`SES` 5. If you are not coming with
-  your own :term:`SES` cluster, this tooling can create an "All-in-one" node
-  for :term:`SES` for you (currently only on OpenStack).
+* The :term:`CaaSP` cluster must run :term:`CaaSP` version 3.
+  :term:`CaaSP` must be updated to its latest 3 version.
```
Comment (jsuchome) [inline doc/source/deploy-guide/overview.rst]: I would write that it is `CaaSP` version 3 and it requires all maintenance updates installed on all its nodes

File: doc/source/deploy-guide/prepare-localhost.rst [hunk 2]
```diff
@@ -138,28 +194,30 @@ the following example if you are running on engcloud (SUSE employees):
...
        region_name: CustomRegion
        auth:
-         auth_url: https://engcloud.prv.suse.net:5000/v3
+         auth_url: https://keystone_url/v3
          username: foctodoodle # your username here
          password: my-super-secret-password # your password here or add it into secure.yaml
          project_name: cloud
          project_domain_name: default
-         user_domain_name: ldap_users
+         user_domain_name: ldap_users # this is just an example, adapt to your needs
```
Comment (jsuchome) [inline doc/source/deploy-guide/prepare-localhost.rst]: All values are examples, right? I'm not sure if it makes sense to specify right here

File: doc/source/deploy-guide/prepare-localhost.rst [hunk 3]
```diff
@@ -138,28 +194,30 @@ the following example if you are running on engcloud (SUSE employees):
...
        identity_api_version: 3
-       cacert: /usr/share/pki/trust/anchors/SUSE_Trust_Root.crt.pem
    ansible:
      use_hostnames: True
      expand_hostvars: False
      fail_on_errors: True
 
 SUSE Employees, you can access the engcloud web UI at https://engcloud.prv.suse.net/.
-For more information, see https://wiki.microfocus.net/index.php/SUSE/ECP.
+For more information on how to set up your `clouds.yaml`, see
```
Comment (jsuchome) [inline doc/source/deploy-guide/prepare-localhost.rst]: "set up your `clouds.yaml` for the usage with ECP"

File: doc/source/deploy-guide/configure-environment.rst [hunk 4]
```diff
@@ -130,18 +132,16 @@ For example:
...
    socok8s_dcm_vip: "192.168.51.35"
 
-Configure certificates for your own registry
---------------------------------------------
-
-If you want to run developer mode, and want to bring your own registry's SSL
-certificates, you can define, in your extra vars:
-
-.. code-block:: yaml
+Provide a kubeconfig file
```
Comment (jsuchome) [inline doc/source/deploy-guide/configure-environment.rst]: This is necessary to implement, but not yet supported by current codebase.

Comment (jsuchome) [reply doc/source/deploy-guide/configure-environment.rst]: Actually I was wrong here, sorry

**Pull request merged**